generate_news_articles.py

```diff
@@ -69,6 +87,6 @@ def phrase_generation(phrases, args):
             parser.error("please provide at least one valid method for generating news with phrases")
         phrase_generation(args.phrases.split(","), args)
     elif args.queries and not args.phrases:
-        if not args.grover and not args.gpt3 and not args.gpt2:
+        if not args.grover and not args.gemini and not args.gpt4 and not args.gpt3 and not args.gpt2:
```

Comment:
here we could also do `if not all([args.grover, args.gemini, ...])` but if it doesn't bother you I'm also okay leaving it like this :)